doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -695,6 +693,11 @@ Libraries for networking
 
   * Updated by adding a missing CONFIG prefix.
 
+* :ref:`lib_downloader` library:
+
+  * Fixed an issue where HTTP download would hang if the application had not set the socket receive timeout and data flow from the server stopped.
+    The HTTP transport now sets the socket receive timeout to 30 seconds by default.
```

Comment:
I got a bit confused that we have a default timeout of 30s but the example shows 60s. Yes, you can do that but I'm thinking if 30s is enough. I don't know how long the timeout would need to be in a really poor network condition.

Reply:
30s is what download_client has so that makes sense to keep the functionality. And this doesn't mean the operation fails but there are some retries.
Let's just keep these as they are.